[IMP] Time Off - Update main doc (#19142)
odoo/documentation: content/applications/hr/time_off.rst

========

Odoo's **Time Off** application serves as a centralized hub for all time-off-related information.
This application manages requests, balances, allocations, approvals, and reports. The following

> documents explain each aspect of the **Time Off** application's various aspects:

Reviewer: ```suggestion
documents explain each aspect of the **Time Off** application:
```

.. cards::

   .. card:: Time off types
      :target: time_off/time_off_types

      Before time off can be requested, the various time off types must be configured.

   .. card:: Public holidays
      :target: time_off/public_holidays

      Configure all public holidays so employees receive the correct days off according to local

>       and national laws.

Reviewer: ```suggestion
      Configure all public holidays so employees receive the correct days off according to local and national laws.
```
Early line break

   .. card:: Allocations
      :target: time_off/allocations

      Some time off types require management to grant time off to employees before it is available

>       to request.

Reviewer: ```suggestion
      Some time off types require management to grant time off to employees before they can request it.
```

   .. card:: Accrual plans
      :target: time_off/accrual_plans

      If time off is earned as employees work, an accrual plan must be configured.

   .. card:: Mandatory days
      :target: time_off/mandatory_days

      Configure mandatory days to prevent employees from requesting time off on a day their
      attendance is required.

   .. card:: Request time off
      :target: time_off/request_time_off

      Before employees take time off, they must submit a time off request.

   .. card:: My time
      :target: time_off/my_time

      Manage personal time off balances, allocations, and time off requests.

   .. card:: Management
      :target: time_off/management

      Review time off and allocation requests from their respective dashboards.

   .. card:: Reporting
      :target: time_off/reporting

      View a summary of all time off, by employee, time off type, or balance.

Overview
========

To view a color-coded schedule of the user's time off and the team they manage, navigate to

> :menuselection:`Time Off app --> Overview`. This loads the *All Time Off* dashbaord, which

Reviewer: ```suggestion
:menuselection:`Time Off app --> Overview`. This loads the *All Time Off* dashboard, which
```

> displays a Gantt view of all time off requests for the user and their team, grouped by employee.

Reviewer: ```suggestion
:menuselection:`Time Off app --> Overview`. This loads the *All Time Off* dashbaord, which displays a Gantt view of all time off requests for the user and their team, grouped by employee.
```
Early line break

:guilabel:`Month` :icon:`fa-caret-down` button to reveal a drop-down menu. Then, select either
:guilabel:`Day`, :guilabel:`Week`, :guilabel:`Month`, :guilabel:`Quarter`, :guilabel:`Year`, or a
custom time period.

To navigate forward or backward in time in the selected increment (:guilabel:`Month`,
:guilabel:`Week`, etc.), click the :icon:`oi-arrow-left` :guilabel:`(left arrow)` or
:icon:`oi-arrow-right` :guilabel:`(right arrow)` buttons to move either forward or backward in that
specified amount of time. For example, if :guilabel:`Month` is selected, the arrows adjust the view
by one month. To return to a view containing the current day, click the :guilabel:`Today` button at
any time.

Team members are listed alphabetically by first name, and display their time off requests with a
status of *Approved* or *Second approval*. Each employee is color-coded. The employee's color is
selected at random, and does *not* correspond to the type of time off requested.

The status of the time off is represented by the color detail of the request, either appearing solid
(*Approved*) or striped (*Second Approval*).

> Each time off entry displays the employee name, time off type, and amount of time requested.

Reviewer: ```suggestion
Each time off entry displays the employee name, time off type, and the amount of time requested.
```

.. note::
   If the time off entry is small and cannot display any text, the entry appears as a solid color
   with no text. All other entries display the text, but the text may be cut off based on the size

>    of the request.

Reviewer: ```suggestion
   with no text. All other entries display the text, but it may be cut off depending on the request size.
```

At the bottom of the calendar, in the :guilabel:`Total` line, a bar graph shows how many employees
are projected to be out on any given day. The number on each individual bar represents the number of

> employees out for those highlighted days.

Reviewer: ```suggestion
are projected to be out on any given day. The number on each bar represents the number of employees out on the highlighted days.
```

      Review time off and allocation requests from their respective dashboards.

   .. card:: Reporting
      :target: time_off/reporting

      View a summary of all time off, by employee, time off type, or balance.

Overview
========

To view a color-coded schedule of the user's time off and the team they manage, navigate to
:menuselection:`Time Off app --> Overview`. This loads the *All Time Off* dashboard, which displays
a Gantt view of all time off requests for the user and their team, grouped by employee.

The *All Time Off* dashboard presents three months of data, the previous, current, and following
months, with the current month visible by default. To change the time period displayed, click the
:guilabel:`Month` :icon:`fa-caret-down` button to reveal a drop-down menu. Then, select either
:guilabel:`Day`, :guilabel:`Week`, :guilabel:`Month`, :guilabel:`Quarter`, :guilabel:`Year`, or a
custom time period.

To navigate forward or backward in time in the selected increment (:guilabel:`Month`,
:guilabel:`Week`, etc.), click the :icon:`oi-arrow-left` :guilabel:`(left arrow)` or
:icon:`oi-arrow-right` :guilabel:`(right arrow)` buttons to move either forward or backward in that

> specified amount of time. For example, if :guilabel:`Month` is selected, the arrows adjust the view

Reviewer 2: ```suggestion
:icon:`oi-arrow-right` :guilabel:`(right arrow)` buttons. For example, if :guilabel:`Month` is
selected, the arrows adjust the view by one month. To return to a view containing the current day,
```
I have to fix this in two separate comments, because Github won't let me fix it otherwise. Basically, the sentence contained two instances of "To navigate forward or backward." I think the sentence could be simplified.

by one month. To return to a view containing the current day, click the :guilabel:`Today` button at

> any time.

Reviewer 2: ```suggestion
click the :guilabel:`Today` button at any time.
```
The second part of the same fix.

Team members are listed alphabetically by first name, and display their time off requests with a

> status of *Approved* or *Second approval*. Each employee is color-coded. The employee's color is

Reviewer 2: Out of curiosity, is the capitalization "Second approval" or "Second Approval"? You use both in both paragraphs.

selected at random, and does *not* correspond to the type of time off requested.

The status of the time off is represented by the color detail of the request, either appearing solid
(*Approved*) or striped (*Second Approval*).

Each time off entry displays the employee name, time off type, and the amount of time requested.

.. note::
   If the time off entry is small and cannot display any text, the entry appears as a solid color
   with no text. All other entries display the text, but it may be cut off depending on the request
   size.

At the bottom of the calendar, in the :guilabel:`Total` line, a bar graph shows how many employees
are projected to be out on any given day. The number on each bar represents the number of employees
out on the highlighted days.

Click on a time off entry to open a *Time Off Request* pop-up window, containing a summary of the

> time off entry. The :guilabel:`Employee`, :guilabel:`Time off Type`, :guilabel:`Dates`, and notes

Reviewer 2: Just to verify, is the capitalization in the UI "Time off Type"? Or should the "O" be capitalized?

Author: Oops - I missed the capital O - you are correct!
